script/testing/util/test_server.py

```diff
@@ -9,12 +9,13 @@
 import errno
 
 from util import constants
+from util.test_case import TestCase
 from util.common import *
 from util.constants import LOG
 
-
-class TestServer:
+class TestServerV2:
```

Comment:
Should we use the original TestServer name?

Reply:
right! I forgot